MS Teams State storage in File

pankaj-a-khandelwal-db opened the pull request.

vaibhav-db committed:
MS Teams file state storage

vaibhav-db committed:
MS teams file storage test cases

robmoffat commented on libs/teams/teams-chat-workflow-spring-boot-starter/src/test/java/org/finos/springbot/teams/state/FileStateStorageTest.java:
```diff
@@ -0,0 +1,87 @@
+
+	@Override
+	@Test
+	public void testStoreWithTagDates() throws IOException {
```
Why is this overridden from the base class?

robmoffat replied:
maybe remove this method completely - it will just inherit from the base class

vaibhav-db committed:
Refactor testcases

vaibhav-db committed:
Refactor file stage storage

vaibhav-db committed:
Refactor file stage storage

vaibhav-db committed:
Refactor file stage storage

vaibhav-db committed:
revert the azure state storage testcase

vaibhav-db committed:
Removed test cases method, it is already present in parent class

robmoffat commented:
I've merged my PR, Pankaj

vaibhav-db committed:
Pull request #23: Spring bot master

Merge in SYMPHONYP/symphony-java-toolkit from spring-bot-master to spring-bot-master-db

* commit '3f59af66b7a8766c6670806f4e72a793043cb5a2':
  Update SECURITY.md
  Update SECURITY.md
  Update SECURITY.md
  Added version number for spring plugin
